korman/nodes/node_messages.py

```diff
@@ -877,3 +877,35 @@ def convert_message(self, exporter, so):
         msg.BCastFlags |= (plMessage.kPropagateToModifiers | plMessage.kNetPropagate)
         msg.surface = footstep_surface_ids[self.surface]
         return msg
+
+
+class PlasmaWarpMsgNode(PlasmaMessageNode, bpy.types.Node):
+    bl_category = "MSG"
+    bl_idname = "PlasmaWarpMsgNode"
+    bl_label = "Warp"
+
+    pos_object = PointerProperty(name="Position",
+                                 description="Object defining the target position",
+                                 type=bpy.types.Object)
+
+    output_sockets = OrderedDict([
+        ("receivers", {
+            "text": "Send To",
+            "type": "PlasmaWarpMessageSocket",
+            "valid_link_sockets": {"PlasmaWarpMessageSocket", "PlasmaNodeSocketInputGeneral"},
```

Comment:
`"PlasmaNodeSocketInputGeneral"`?

Reply:
After the explanation in IRC, it might be better to use `valid_link_nodes` instead and specify the expected node type.